tracker/tracker_debugger_api.yaml

```diff
@@ -1318,3 +1374,912 @@ components:
           description: Google Click ID
           type: string
           example: 1237abcdb
+    Log:
+      description: A tracking log
+      required:
+        - id
+        - time
+        - ip
+        - scheme
+        - method
+        - headers
+        - query_params
+      type: object
+      properties:
+        id:
+          description: UUID of the log
+          type: string
+          format: uuid
+          example: 76d1b36e-c2d5-44e8-ad60-028cb0bf17f6
+        time:
+          description: 'Time when the request was received by a tracker (in UTC)'
+          type: string
+          format: date-time
+          example: '2020-02-01T00:00:00.000000000Z'
+        ip:
+          description: 'Tracker client IP (real value can be provided in headers or `cip` parameter)'
+          type: string
+          format: ip
+          example: 75.13.13.87
+        host:
+          description: 'A host under which app is available'
+          type: string
+          example: piwikpro.tld
+        scheme:
+          description: 'Scheme used for the request'
+          type: string
+          example: http
+        method:
+          description: 'Method used for the request'
+          type: string
+          example: GET
+        headers:
+          description: Request HTTP headers
+          type: array
+          items:
+            type: array
+            items:
+              type: string
+          example:
+            - - 'x-forwarded-for'
+              - '172.18.0.15'
+            - - 'accept-language'
+              - 'en-US,en;q=0.9'
+            - - 'user-agent'
+              - 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/66.0.3359.117 Safari/537.36'
+        query_params:
+          description: 'Tracking request parameters'
+          type: object
+          $ref: '#/components/schemas/TrackingParameters'
```

Comment:
Will query_params be deserialized or still in a string?

Reply:
As documentation states: object (meaning: deserialized to JSON).